support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -474,6 +474,7 @@ Interpolator:
   Verbosity: Silent
 
 ApparentHorizons:
+  LMax: 33
```

Comment:
Question: should each AH have a separate LMax? (can be done by templating the tag on the `HorizonMetavars`, or by adding LMax to `ApparentHorizonOptions`)

Second question: While no criteria are configured here, should `LMax` be smaller (e.g. the initial guess LMax) so we don't write so many padded zeros?

Reply:
Ok probably padding with zeros is not a big deal. We currently observe each horizon every 0.5M. LMax=33 means about 10^3 coefficients. That's 80 MB of data for 5000 M of evolution. Might be reduced by H5 compression. Let's see if this becomes a problem.